Test stash

sanlengren opened the pull request:
this is to test the pull request could be review / approve / merged

sanlengren committed:
0020 - updated github rule setting apply to admin

sanlengren committed:
0020 - Commit in branch to test pull request

sanlengren committed:
Merge branch 'master' into test_stash

# Conflicts:
#	src/test/com/kelvin/testbase/TestGitBehavior.java

kelvin2backup reviewed (changes requested):
code review not pass!

sanlengren committed:
Comitt again

sanlengren committed:
Test commit before review would show up in the pull requst.

sanlengren committed:
Pilot - 30 lotss of commit
Current focus is the thread. 
This commit show a sample about 2 thread manipuate 1 object. in this
practice I learn why arraylist is not thread safety. next step, I would
abstract this object out to wrap up the arraylist and make this lock
when thread call it.
